challenge_03_loops-conditionals/loops-conditionals/conditionals/exercise_02.js

```diff
@@ -0,0 +1,63 @@
+/* 
+   Spot the bug 🐞
+   Should print: "Most places consider you an adult."... but it doesn't, why?
+
+   Add a comment explaining what is happenning and how to fix it
+
+const age = 20;
+
+if (age < 3) {
+  console.log("You're just a baby!");
+} else if (age >= 3) {
+  console.log("You are in elementary school, kid.");
+} else if (age >= 13) {
+  console.log("Nice, you're a teenager!");
+} else if (age >= 18) {
+  console.log("Most places consider you an adult.");
+} else {
+  console.log("What? How did this happen!?");
+}
+This program has a bug in the main code. First of all, if we follow the code
+as we got it and we know that JS is a language that executes sequentially.
+So, when our variable 'age' enters the first conditional that fits it, 
+it will exit the program; that's why if you execute it on your console 
+will show you "you are in elementary school".
+To fix the code, we will need to add a maximum and a minimum value in 
+every condition, in order to place our variable in the condition that suits it perfectly.
+The code should be like this:
+const age = 20;
+if (age < 3) {
+  console.log("You're just a baby!");
+} else if (age >= 3 && age < 13) {
+  console.log("You are in elementary school, kid.");
+} else if (age >= 13 && age < 18) {
+  console.log("Nice, you're a teenager!");
+} else if (age >= 18) {
+  console.log("Most places consider you an adult.");
+} else {
+  console.log("What? How did this happen!?");
+}
+*/
+let message;
+let userAge = 20;
+
+const howOldAreYou = function (age) {
+  if (typeof age !== "number") {
+    return "What? How did this happen!?";
+  }
+  if (age >= 18) {
+    return "Most places consider you an adult.";
+  }
+  if (age >= 13) {
+    return "Nice, you're a teenager!";
+  }
+  if (age >= 3) {
+    return "You are in elementary school, kid.";
+  } else {
```

Comment:
Second rule of Object Calisthenics:  **Don’t Use The ELSE Keyword** ([doc ↗](https://github.com/DAW-EPIA-MAT/clean-code/blob/main/clean-code/OBJECT_CALTISTHENICS.md#-dont-use-the-else-keyword))

FIX> refactor method to avoid ELSE (take a look to examples I provided last week. You can find them attached in our chat)